vendor/github.com/cilium/tetragon/api/v1/tetragon/codegen/eventchecker/eventchecker.pb.go

```diff
@@ -5077,6 +5138,12 @@ func (checker *KprobeArgumentChecker) WithModuleArg(check *KernelModuleChecker)
 	return checker
 }
 
+// WithLinuxBinprmArg adds a LinuxBinprmArg check to the KprobeArgumentChecker
+func (checker *KprobeArgumentChecker) WithLinuxBinprmArg(check *KprobeLinuxBinprmChecker) *KprobeArgumentChecker {
```

Comment:
Is this consistent with other tetragon constructs?

Reply:
This is generated code, so there's no point in editing it. This is something I'd like to change in the future, why are we checking in generated code at all? The code generation should take place as a job in their CI.

Anything that ends in `.pb.go` or `.pb.json` is generated.